pages/installation/os/centos.rst

```diff
@@ -82,6 +81,18 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl enable graylog-server.service
   $ sudo systemctl start graylog-server.service
 
+SELinux Information
+----------------------
+
+If you run your System with SELinux you need to take care of the following
+
+- enable http on all ports  ``sudo setsebool -P httpd_can_network_connect 1``
+- if the above does not follow your security rules
+		- Graylog API ``sudo semanage port -a -t http_port_t -p tcp 12900``
+		- elasticsearch ``sudo semanage port -a -t http_port_t -p tcp 9200``
```

Comment:
Please use two spaces instead of tabs.

Also: https://www.youtube.com/watch?v=SsoOG6ZeyUI